### PR TITLE
chore: fixes for stale_msp_after_node_power_failure_test

### PR DESCRIPTION
**Pull request by blaisedias**
- after applying the node selectori
 - pause for 1 min to allow the pods to migrate
 - ascertain that migration was successful by readiness check
- migrate the csi-controller
 - strictly speaking this isn't required but mimics moac deployment

### CODE REVIEW

File: src/stale_msp_after_node_power_failure/stale_msp_after_node_power_failure_test.go [hunk 1]
```diff
@@ -109,6 +108,11 @@ func (c *nodepowerfailureConfig) staleMspAfterNodePowerFailureTest() {
 
 		k8stest.ApplyNodeSelectorToDeployment("msp-operator", common.NSMayastor(), "kubernetes.io/hostname", coreAgentNodeName)
```
Comment (chriswldenyer) [inline src/stale_msp_after_node_power_failure/stale_msp_after_node_power_failure_test.go:109]: Could do with a common here to explain why this is being done.

**Review by chriswldenyer — APPROVED** [review]
LGTM, one comment

**Review by payes — APPROVED** [review]

**Comment by blaisedias** [comment]
bors merge

**Comment by blaisedias** [comment]
bors merge

**Pull request merged**